### PR TITLE
Add back FAB backends used in webserver config

### PR DESCRIPTION
**Pull request by jedcunningham**
Every Airflow today imports the FAB auth type from us, so maintain them for backcompat.

### CODE REVIEW

File: airflow/www/fab_security/manager.py [hunk 1]
```diff
@@ -63,6 +63,35 @@
...
+
+def __getattr__(name: str):
+    # PEP-562: Lazy loaded attributes on python modules
+    path = __lazy_imports.get(name)
+    if not path:
+        raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+
+    import warnings
+
+    warnings.warn(
```
Comment (jedcunningham) [inline airflow/www/fab_security/manager.py]: I'd also be open to not deprecating importing from this module. Thoughts @vincbeck?

Comment (vincbeck) [reply airflow/www/fab_security/manager.py]: Yep, I am good with that

**Review by potiuk — APPROVED** [review]

File: airflow/www/fab_security/manager.py (hunk 1 above)
Comment (potiuk) [reply airflow/www/fab_security/manager.py]: yes

File: airflow/www/fab_security/manager.py [hunk 2]
```diff
@@ -63,6 +63,35 @@
...
+        f"Importing {name!r} directly from {__name__!r} is deprecated and "
+        f"will be removed in the future. Please import it from '{path}.{name}'.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+
```
Comment (jedcunningham) [inline airflow/www/fab_security/manager.py]:
```suggestion
```

**Review by vincbeck — APPROVED** [review]

**Pull request merged**